website/versioned_docs/version-25.x/MockFunctions.md

```diff
@@ -17,41 +17,49 @@ function forEach(items, callback) {
     callback(items[index]);
   }
 }
+module.exports = forEach;
```

Comment:
One more thought. What about using ESM syntax here? The examples below written in ESM: https://jestjs.io/docs/next/mock-functions#mocking-modules

Reply:
[ESM syntax](https://tc39.es/ecma262/#sec-exports) is then `export forEach` ?

Reply:
It is:

```js
export function forEach(items, callback) {
  // etc ...
```

And instead of `const forEach = require('./foreach');`:

```js
import {forEach} from './forEach';
```